docs/For User/Account keys/02 Multisig.md

```diff
@@ -0,0 +1,256 @@
+
+## Generate a Multisig key
+
+```linux
+imversed keys add --multisig=name1,name2,name3[...] --multisig-threshold=K new_key_name
+```
+
+`K` is the minimum number of private keys that must have signed the transactions that carry the public key's address as signer.
+
+The `--multisig` flag must contain the name of public keys that will be combined into a public key that will be generated and stored as `new_key_name` in the local database. All names supplied through `--multisig` must already exist in the local database.
+
+Unless the flag `--nosort` is set, the order in which the keys are supplied on the command line does not matter, i.e. the following commands generate two identical keys:
+
+```linux
+imversed keys add --multisig=p1,p2,p3 --multisig-threshold=2 multisig_address
+imversed keys add --multisig=p2,p3,p1 --multisig-threshold=2 multisig_address
+```
+Multisig addresses can also be generated on-the-fly and printed through the which command:
+
+```linux
+imversed keys show --multisig-threshold=K name1 name2 name3 [...]
+```
+
+## Signing a transaction
+
+### Step 1: Create the multisig key
+
+Let's assume that you have `test1` and `test2` want to make a multisig account with `test3`.
+
+First import the public keys of `test3` into your keyring.
+
+```linux
+imversed keys add \
+    test3 \
+    --pubkey=imversedpub1addwnpepqgcxazmq6wgt2j4rdfumsfwla0zfk8e5sws3p3zg5dkm9007hmfysxas0u2
+```
+
+Generate the multisig key with 2/3 threshold.
+
+```linux
+imversed keys add \
+    multi \
+    --multisig=test1,test2,test3 \
+    --multisig-threshold=2
+```
+
+You can see its address and details:
+
+```linux
+imversed keys show multi
+
+- name: multi
+  type: multi
+  address: imversed1e0fx0q9meawrcq7fmma9x60gk35lpr4xk3884m
+  pubkey: imversedpub1ytql0csgqgfzd666axrjzq3mxw59ys6yqcd3ydjvhgs0uzs6kdk5fp4t73gmkl8t6y02yfq7tvfzd666axrjzq3sd69kp5usk492x6nehqjal67ynv0nfqapzrzy3gmdk27la0kjfqfzd666axrjzq6utqt639ka2j3xkncgk65dup06t297ccljmxhvhu3rmk92u3afjuyz9dg9
+  mnemonic: ""
+  threshold: 0
+  pubkeys: []
+
+```
+
+Let's add 10 imversed to the multisig wallet:
+
+```linux
+imversed tx bank send \
+    test1 \
+    imversed1e0fx0q9meawrcq7fmma9x60gk35lpr4xk3884m \
+    10000000000000000000aimv \
+    --chain-id=imversed_9000-4 \
+    --gas=auto \
+    --fees=1000000aimv \
+    --broadcast-mode=block
+```
+
+### Step 2: Create the multisig transaction
+We want to send 5 IMV from our multisig account to `imversed1rgjxswhuxhcrhmyxlval0qa70vxwvqn2e0srft`.
+
+```linux
+imversed tx bank send \
+    imversed1rgjxswhuxhcrhmyxlval0qa70vxwvqn2e0srft \
+    imversed157g6rn6t6k5rl0dl57zha2wx72t633axqyvvwq \
+    5000000000000000000aimversed \

```

Comment:
Change 5000000000000000000**aimversed** to 5000000000000000000**aimv**

Reply:
Updated